apps/spruce/src/pages/version/Tabs/Tasks/index.tsx

```diff
@@ -82,36 +80,17 @@ const Tasks: React.FC<Props> = ({ taskCount, versionId }) => {
   const { count = 0, data: tasksData = [] } = tasks || {};
 
   return (
-    <TableWrapper
-      controls={
-        <TableControl
-          filteredCount={count}
-          label="tasks"
-          // @ts-expect-error: FIXME. This comment was added by an automated script.
-          limit={limit}
-          onClear={clearQueryParams}
-          onPageSizeChange={(l) => {
-            versionAnalytics.sendEvent({
-              name: "Changed page size",
-              "page.size": l,
-            });
-          }}
-          // @ts-expect-error: FIXME. This comment was added by an automated script.
-          page={page}
-          totalCount={taskCount}
-        />
-      }
-      shouldShowBottomTableControl={tasksData.length > 10}
-    >
-      <PatchTasksTable
-        // @ts-expect-error: FIXME. This comment was added by an automated script.
-        isPatch={isPatch}
-        loading={tasksData.length === 0 && loading}
-        // @ts-expect-error: FIXME. This comment was added by an automated script.
-        sorts={sorts}
-        tasks={tasksData}
-      />
-    </TableWrapper>
+    <VersionTasksTable
+      clearQueryParams={clearQueryParams}
+      filteredCount={count}
+      isPatch={isPatch ?? false}
+      limit={limit ?? 0}
+      loading={tasksData.length === 0 && loading}
+      page={page ?? 0}
+      // @ts-expect-error: Expects TaskTableInfo which overlaps with this type
```

Comment:
Would it be possible to fix the typing in https://github.com/evergreen-ci/ui/blob/02acfa1bd2fa369fd75c379f9a78701b4455f39f/apps/spruce/src/components/TasksTable/types.ts so that it reflects the real types? (And maybe file a ticket to get rid of the null or undefined stuff in evergreen)?